test/integration/features/cmd-addons.feature

```diff
@@ -8,9 +8,12 @@ Feature: Addons command and its subcommands
       And executing "minishift add-ons" fails
       And executing "minishift add-on" fails
 
-  @minishift-only
-  Scenario: Default add-ons are installed but not enabled by default
-  Note: default addons were installed during previous commands automatically.
+ Scenario: Installing default add-ons manually
+     When executing "minishift addons install --defaults" succeeds
+     Then stdout should contain
+      """
+      Default add-ons anyuid, admin-user, xpaas, registry-route installed
+      """
```

Comment:
@agajdosi @amitkrout FYI The add-on list default behavior is changing with this PR.

Reply:
@LalatenduMohanty Sorry I am replying late. This change is correct, in other features we use addons only in basic and there is seems that this PR won't affect the test.